### DISCUSSION TITLE
dodgy ms2130?? dont see 4k only 1080p. help please

### DISCUSSION THREAD

**Discussion by valtoree** [awawa-dev/HyperHDR#892] [Q&A] [answered]
hi everyone,
bought ms2130 from this ad https://www.aliexpress.com/item/1005004990708725.html
but on the hyperhdr app, video modes only go up to 1080p.
so i am confused.
I did read before and i did confirm that usb3 gives YUYV and usb2 gives mjepg, and i went ahead and installed custom LUT.
everything is working, but i assume i should have 4k video modes on device info (attached).
do i have a dud?
i can still claim refund at this stage but want to see if i haven't done anything wrong,
thanks for reading
![Mon-01-Jul-24_13-49-43](https://github.com/awawa-dev/HyperHDR/assets/70986369/a8192468-da2d-4431-bff9-bc2e4297c71e)

**Comment by awawa-dev** [comment, marked as answer]
Hi
Everything is fine. As you can read in the thread about ms2130, there is one Chinese firmware (which you can upload to ms2130 if you want, but I don't recommend it) that offers 4k capture but at the expense of quality. Moreover, in HyperHDR you don't even need 1080p, just a lower resolution, which is completely sufficient. The higher the resolution, the more power is needed to process frames.

**Reply by valtoree** [reply]
thanks for reply :)
i saw your old comment (https://github.com/awawa-dev/HyperHDR/discussions/499#discussioncomment-7812868)
and was concerned about it.
I confirmed the input of the device by my pc gpu resolution settings and could see 4k input like your comment here (https://github.com/awawa-dev/HyperHDR/discussions/499#discussioncomment-4961638)
thanks again.
